internal/net/ssl/ssl.go

```diff
@@ -452,11 +457,6 @@ func FullChainCert(in string, fs file.Filesystem) ([]byte, error) {
 		return nil, err
 	}
 
-	certs, err = certUtil.AddRootCA(certs)
-	if err != nil {
-		return nil, err
-	}
-
```

Comment:
why is this not needed anymore?

Reply:
To trust the certs from the linux image.